WordPress/openverse
# Set the background color on `<option>` to fix readability issue

## Fixes



Fixes #5277 by @dhruvkb

## Description



This PR adds a `bg-overlay` class to the `<option>` elements so that they will be readable in both dark and light modes. Prior to this, they were inheriting the `bg-tx` from the parent `<select>` and rendering a white background in dark mode, making the light text unreadable.

## Screenshot

<img width="1515" alt="image" src="https://github.com/user-attachments/assets/0b26b5e3-d289-4b1a-8f12-97debc3423ad" />

## Testing Instructions


> [!NOTE] 
> You need to test this on Windows. It'll be nice if you have a Windows machine, but you can also test using [BrowserStack Live (with their Local plugin)](https://www.browserstack.com/docs/live/local-testing).

0. Check out the PR and run the dev server.
1. Switch between dark and light theme and try to see the language selector and theme dropdowns.
2. The text should always be readable.

## Checklist


      PRs) or the media properties generator (`ov just catalog/generate-docs media-props`
      for the catalog or `ov just api/generate-docs` for the API) where applicable.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
